lib/instrument/index.js

```diff
@@ -127,6 +128,9 @@ class InstrumentService extends BaseServiceSocket {
 
   async _callChannel (sync, channelCode, selector, ...auxiliaries) {
     const identifier = this._nextIdentifier;
+    if (auxiliaries.length === 1 && auxiliaries[0] instanceof DTXMessageAuxBuffer) {
+      auxiliaries = auxiliaries[0];
```

Comment:
It is usually not a good practice to mutate function argument values. How about changing it to 
```
this._encoder.write({
  sync, 
  channelCode, 
  selector, 
  auxiliaries.length === 1 && auxiliaries[0] instanceof DTXMessageAuxBuffer ? auxiliaries[0] :  auxiliaries, 
  identifier
});
```

Reply:
This is actually a `key: value` with same name in an object.
Will following code be ok?
```
this._encoder.write({
  sync, 
  channelCode, 
  selector, 
  auxiliaries: (auxiliaries.length === 1 && auxiliaries[0] instanceof DTXMessageAuxBuffer) ? auxiliaries[0] :  auxiliaries, 
  identifier
});
```

Reply:
yes, it should be ok. Thanks for correcting me